src/main/java/org/takes/rq/RqMultipart.java

```diff
@@ -288,11 +310,13 @@ private Request make(final byte[] boundary) throws IOException {
          * Copy until boundary reached.
          * @param target Output file channel
          * @param boundary Boundary
+         * @param body Origin request body
          * @throws IOException If fails
          * @checkstyle ExecutableStatementCountCheck (2 lines)
          */
         private void copy(final WritableByteChannel target,
-            final byte[] boundary) throws IOException {
+            final byte[] boundary,  final  ReadableByteChannel body)
```

Comment:
@2686747 Double whitespace here again, between `,` and `final`, and between `final` and `ReadableByteChannel`